src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -148,6 +148,12 @@ public enum CurrentState
         /// </summary>
         public const long MinimumStakingCoinValue = 10 * Money.CENT;
 
+        /// <summary>When splitting a big utxo, this is the number of smaller utxos we divide it into.</summary>
+        internal const int SplitFactor = 8;
+
+        /// <summary>Minimum value of a split utxo we are aiming for (after splitting it into <c><seealso cref="SplitFactor" /></c> equal parts).</summary>
```

Comment:
`seealso` is not used like that
if you want to use a reference to something existing, you just use `see` (and without `c`)

`seealso` is a separated section and usually lives outside of other sections - https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#ltseealsogt

so seealso is used outside of sentence and is just link - look at https://msdn.microsoft.com/en-us/library/dd78zt0c%28v=vs.110%29.aspx#Anchor_5 for example - very bottom of page, there is See Also section